boards/unphone8.json

```diff
@@ -0,0 +1,41 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32s3_out.ld"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DUNPHONE_SPIN=8",
+      "-DARDUINO_USB_CDC_ON_BOOT=0",
+      "-DARDUINO_RUNNING_CORE=1",
+      "-DARDUINO_EVENT_RUNNING_CORE=1"
+    ],
+    "f_cpu": "240000000L",
+    "f_flash": "80000000L",
+    "flash_mode": "dio",
+    "mcu": "esp32s3",
+    "variant": "unphone8"
```

Comment:
I'm afraid this won't work as there is no such variant in the official Arduino core for ESP32 https://github.com/espressif/arduino-esp32/tree/master/variants . Shouldn't it be added there in the first place?

Reply:
You're right! But we've submitted a PR there a few days ago:
https://github.com/espressif/arduino-esp32/pull/7058
In the meantime our fork provides that definition:
https://github.com/garethhcoleman/arduino-esp32